bitnami/elasticsearch/templates/data-statefulset.yaml

```diff
@@ -216,13 +216,17 @@ spec:
 {{- if not .Values.data.persistence.enabled }}
         - name: "data"
           emptyDir: {}
+{{- else if .Values.data.persistence.existingClaim }}
+        - name: "data"
+          persistentVolumeClaim:
+            claimName: {{ .Values.data.persistence.existingClaim }}
 {{- else }}
   volumeClaimTemplates:
     - metadata:
         name: "data"
         {{- if .Values.data.persistence.annotations }}
         annotations: {{- toYaml .Values.data.persistence.annotations | nindent 10 }}
-        {{- end }}
+        {{- end }}        
```

Comment:
It looks like some trailing spaces were added, could you remove them?

Reply:
removed trailing spaces, added documentation to values.yaml, values-production.yaml and README.md